CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Changelog
 * Fabric Path
   * fabricpath_global (@dcheriancisco)
   * fabricpath_topology (@dcheriancisco)
+* Interface
+  * interface_service (@chrisvanheuveln)
+    * `encapsulation_profile_vni`
```

Comment:
It's not a sub-category. I debated listing it like this but wanted some way to indicate that it was needed for vni. I can remove it if you prefer.